docs/src/main/sphinx/connector/redshift.rst

```diff
@@ -77,13 +77,23 @@ Type mapping
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _redshift-sql-support:
+
 SQL support
 -----------
 
-The following SQL statements are not yet supported:
-
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
+The connector provides read access and write access to data and metadata in
+Redshift. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
```

Comment:
Should this be?
```suggestion
statements, the connector supports the following statements:
```

Reply:
See https://github.com/trinodb/trino/pull/8668#discussion_r678538095, leaving as-is for now